autopush/web/message.py

```diff
@@ -9,7 +10,7 @@
 
 class MessageSchema(Schema):
     uaid = fields.UUID()
-    channel_id = fields.UUID(allow_none=True)
+    channel_id = fields.Str(allow_none=True)
```

Comment:
If you really want it to be a str on the way out, the post_load should transform it after it's validated as UUID or None.